unstructured/ingest/connector/dropbox.py

```diff
@@ -0,0 +1,78 @@
+from dataclasses import dataclass
```

Comment:
this will be a separate pr, right? assuming just some extra in-progress stuff in here?

Reply:
Damn. How did that turkey get in there. That file is for another PR.

Reply:
Same with main_for_dropbox.py. I will remove this afternoon!

Reply:
Sweet, thanks!